Disable platform-specific tests.
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -118,16 +118,16 @@
 
 target_link_libraries(dlopen dl)
 
-if (${CMAKE_SYSTEM_NAME} MATCHES Linux)
-	add_executable(recursive-ptrace ${recursive-ptrace_SRCS})
-	add_custom_target (illegal-insn ALL
-  		COMMAND ${CMAKE_C_COMPILER} -nostdlib ${CMAKE_CURRENT_SOURCE_DIR}/assembly/illegal-insn.S -o ${CMAKE_BINARY_DIR}/illegal-insn
-	)
-
-	add_custom_target (fork-32 ALL
-  		COMMAND ${CMAKE_C_COMPILER} -g -m32 ${CMAKE_CURRENT_SOURCE_DIR}/fork/fork.c -o ${CMAKE_BINARY_DIR}/fork-32
-	)
-endif()
+#if (${CMAKE_SYSTEM_NAME} MATCHES Linux)
+#	add_executable(recursive-ptrace ${recursive-ptrace_SRCS})
+#	add_custom_target (illegal-insn ALL
+#  		COMMAND ${CMAKE_C_COMPILER} -nostdlib ${CMAKE_CURRENT_SOURCE_DIR}/assembly/illegal-insn.S -o ${CMAKE_BINARY_DIR}/illegal-insn
+#	)
+#
+#	add_custom_target (fork-32 ALL
+#  		COMMAND ${CMAKE_C_COMPILER} -g -m32 ${CMAKE_CURRENT_SOURCE_DIR}/fork/fork.c -o ${CMAKE_BINARY_DIR}/fork-32
+#	)
+#endif()
 
 target_link_libraries(shared_library_test shared_library)
 target_link_libraries(global-constructors shared_library)
